pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ProducerConfigurationData.java

```diff
@@ -70,6 +70,7 @@
     private MessageRouter customMessageRouter = null;
 
     private long batchingMaxPublishDelayMicros = TimeUnit.MILLISECONDS.toMicros(1);
+    private int batchingPartitionSwitchFrequencyByPublishDelay = 10;
```

Comment:
Add "Micros" to the name so people know the time unit directly by reading the name.